jbcli/jbcli/utils/reload.py

```diff
@@ -22,20 +22,18 @@ def refresh_browser(timeout=None):
     server status
     """
     click.echo('Checking server status...')
-
-    if timeout:
-        time.sleep(timeout)
-
-    def attempt_refresh():
+    for i in range(5):
+        if timeout:
+            time.sleep(timeout)
+        response = {'status_code': None}
```

Comment:
this doesn't seem to be a meaningful line, since it will immediately get overwritten with the result of `get`.

Reply:
Whoops! Meant to remove this. Thanks.